### PR TITLE
blocker.py

### PR DESCRIPTION
**Pull request by ShashiNova**

### CODE REVIEW

**Review by iamwatchdogs — COMMENTED** [review]
@ShashiNova it's recommended to add a `README.md` file.

File: blocker.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                hostfile.seek(0)
+                for host in hosts:
+                    if not any(site in host for site in sites_to_block):
+                        hostfile.write(host)
+                hostfile.truncate()
+            print("Good Time")
+        time.sleep(3)
+
+
+if _name_ == "_main_":
```
Comment (iamwatchdogs) [inline blocker.py:44]: I went through your code as the linting checks have not passed and found this issue. Please fix this syntactical error and test your code once again.

```suggestion
if __name__ == "_main_":
```

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]
@ShashiNova Please make the required changes.

File: blocker.py
Comment (iamwatchdogs) [inline blocker.py:1]: @ShashiNova Please place your project files within a directory/folder named with your project name.

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]
Please make the requested changes both for lining checks _(previous code review)_ and maintain the file structure.

File: blocker.py
Comment (ShashiNova) [reply blocker.py:1]: I have moved the project files into the required folder.

**Review by iamwatchdogs — APPROVED** [review]
The requested changes have been made. Review and Approved by @iamwatchdogs.

**Comment by iamwatchdogs** [comment]
Thank you for contributing @ShashiNova, make sure to check your contributions on [GitHub Pages](https://grow-with-open-source.github.io/Python-Projects/).

**Pull request merged**